src/tools/yaml2json

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
+"""
+Convert YAML file to JSON with identical structure (as a python dict)
+Adapted from https://raw.githubusercontent.com/vasil9v/yaml2json.py/c4cfa408bbab4f1a0cd3661d121237030f6bc0ca/yaml2json.py
+"""
+import sys
+
+# Try to use the system yaml if present, else use the PLSI-shipped yaml.
+try:
+    import yaml
+except ImportError:
+    sys.path.append("src/tools/pyyaml/lib3")
+    import yaml
```

Comment:
I guess the thought is to try to use the system libraries where possible, and fallback on our own libraries if the system one is missing/defective. I believe there is also a small performance gain since the system yaml is probably precompiled but it's likely not significant enough to really matter.